packages/drivers/local-driver/api-extractor/api-extractor.current.json

```diff
@@ -1,5 +1,14 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
 	"extends": "<projectFolder>/../../../common/build/build-common/api-extractor-report.esm.current.json",
-	"mainEntryPointFilePath": "<projectFolder>/lib/public.d.ts"
+	"mainEntryPointFilePath": "<projectFolder>/lib/public.d.ts",
+	// Note: excluding server packages until we can update their tags (and release new server versions)
+	// TODO: remove this override once server dependencies have been updated.
+	"bundledPackages": [
```

Comment:
Note for reviewers: this is an existing workaround in this package's "lint" configurations. The same issue applies to the report configurations now that we're bundling other packages. We can remove these workarounds once we've done a server release and updated dependencies here.